lldb/tools/lldb-rpc-gen/lldb-rpc-gen.cpp

```diff
@@ -46,6 +48,12 @@ static std::string GetServerOutputDirectory() {
   return std::string(Path);
 }
 
+static std::string GetLibraryOutputDirectory() {
+  llvm::SmallString<128> Path(OutputDir.getValue());
```

Comment:
nit: I was wondering if we should make the template argument here be `PATH_MAX` but that's only defined on unix platforms, on windows is another macro called `MAX_PATH` which is hard coded to 260 I believe. May be we should go with that to match the smallest path length.